src/sentry/api/endpoints/accept_project_transfer.py

```diff
@@ -105,12 +102,9 @@ def post(self, request: Request) -> Response:
             return Response({"detail": "Invalid organization"}, status=400)
 
         # check if user is an owner of the organization
-        is_org_owner = OrganizationMember.objects.filter(
-            user__is_active=True,
-            user=request.user,
-            role=roles.get_top_dog().id,
-            organization_id=organization.id,
-        ).exists()
+        is_org_owner = request.access.has_organization_role(
+            role=roles.get_top_dog().id, organization=organization, user_id=request.user.id
```

Comment:
`roles.get_top_dog().id` - this is the ID of the top dog _role_ specifically right?
Like an enum or something?

Or is this the id of the user somehow??

Reply:
Also - if this is an id, can we rename the key `role_id` rather than `role`?

Reply:
the `id` of a role actually is a name like "owner". the `name` of a role is the capitalized version "Owner". from SENTRY_ROLES https://github.com/getsentry/sentry/blob/master/src/sentry/conf/server.py#L1730